package.json

```diff
@@ -58,6 +58,7 @@
         "postcss-nesting": "^10.1.10",
         "prettier": "^2.7.1",
         "react-test-renderer": "^18.2.0",
+        "rtlcss": "^4.0.0",
```

Comment:
Well done @SufyanCS!

Here, I don't think that this package is needed as tailwind already does that by having the document direction to rtl (check `How.jsx` in #46).

Reply:
I apologize for the confusion. I downloaded the package for testing purposes, but I forgot to remove it afterwards. Thank you for reminding me to delete it.